lib/router.js

```diff
@@ -282,11 +282,16 @@ module.exports = class Router extends express.Router {
           res.end('<script>window.close();</script>');
         })
         .catch((err) => {
-          res.status(SYSTEM_ERROR_STATUS).send(err);
+          res.cookie('unifile_error', JSON.stringify(err));
+          res.redirect('/cloud-explorer/handle-errors');
```

Comment:
here you can do:
* redirect with the error in GET
* render the page directly here with res.sendFile and use the info in the query (provided by FTP, dropbox or ...) 
* render the page with a template language and use req.query to know what is the error

Reply:
* or you can render the page with a `res.end('<script>window.opener.getPopupError(.....); window.close();</script>');` and render the error in CE2